### PR TITLE
Formatting check GitHub action

### PR DESCRIPTION
**Pull request by flying-sausages**
Github action to check .sh formatting (and eventually shellcheck too)

### CODE REVIEW

**Comment by flying-sausages** [comment]
Maybe I can set this up to only check the Files Changed? https://github.com/luizm/action-sh-checker/issues/27

File: .github/workflows/sh-checker.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  sh-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run the sh-checker
+        uses: luizm/action-sh-checker@v0.1.8
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SHELLCHECK_OPTS: -e SC1004 # It is posible to exclude some shellcheck warnings.
```
Comment (userdocs) [inline .github/workflows/sh-checker.yml:13]: Do we need this particular exclusion? I know it's in the example action but is it required?

https://github.com/koalaman/shellcheck/wiki/SC1004

Comment (flying-sausages) [reply .github/workflows/sh-checker.yml:13]: Ah lol nope not at all. It should pick up shellcheckrc

**Comment by liaralabs** [comment]
Another shellchecker? Can we potentially trim down since we now have 3 checkers in the repo?

**Comment by flying-sausages** [comment]
> Another shellchecker? Can we potentially trim down since we now have 3 checkers in the repo?

I don't see any other checker being ran under any of the other PRs right now. This is the only one we have here. And with the current implementation, it does `shfmt` checks **only**

**Review by userdocs — APPROVED** [review]

**Pull request merged**